content/blog/2020/07/2020-07-27-machine-learning-plugin-coding-phase2.adoc

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: "Machine Learning Plugin project - Coding Phase 2 blog post"
+tags:
+- machinelearning
+- datascience
+- plugins
+- communitybonding
+- gsoc
+- gsoc2020
+author: loghijiaha
+description: "Coding Phase 2 of Google Summer of Code  2020"
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+image::/images/gsoc/jenkins-gsoc-logo_small.png[align="center"]
+
+Welcome back forks!
```

Comment:
```suggestion
Welcome back folks!
```

Just in case it was a typo :)

Reply:
I also wasn't sure if it was intentional or not :)